src/Installer/Alpha.htm

```diff
@@ -8,6 +8,16 @@
 		<h4>Release Notes (Preview of things to come)</h4>
 		<ul>
 			<li>DEV_VERSION_NUMBER<ol>
+				<li>Chorus: Update documentation of Chorus Hub</li>
+				<li>FLExBridge: Improve developer documentation and update documentation of recent fixes</li>
```

Comment:
Users can see it on the FB download web page:

http://downloads.palaso.org/FlexBridge/index.htm